src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2022,28 +2051,13 @@ public override BoundNode VisitConversion(BoundConversion node)
             {
                 switch (node.ConversionKind)
                 {
-                    case ConversionKind.ExplicitUserDefined:
-                    case ConversionKind.ImplicitUserDefined:
-                        if ((object)node.SymbolOpt != null && node.SymbolOpt.ParameterCount == 1)
-                        {
-                            ReportArgumentWarnings(operand, operandType, node.SymbolOpt.Parameters[0]);
-                        }
-                        break;
-
                     case ConversionKind.AnonymousFunction:
                         if (!node.ExplicitCastInCode && operand.Kind == BoundKind.Lambda)
                         {
                             var lambda = (BoundLambda)operand;
                             ReportNullabilityMismatchWithTargetDelegate(operand.Syntax, targetType.GetDelegateType(), lambda.Symbol);
                         }
                         break;
-
-                    case ConversionKind.MethodGroup:
```

Comment:
>case ConversionKind.MethodGroup: [](start = 20, length = 32)

It is not obvious why it is no linger needed to handle this case.

Reply:
Handled in `ApplyConversion`.

---
In reply to: [183861811](https://github.com/dotnet/roslyn/pull/26356#discussion_r183861811) [](ancestors = 183861811)